Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM rocker/shiny:4.0.0
+FROM rocker/shiny:latest
```

Comment:
perhaps you can specify the shiny version you have used, using `latest` could break in a couple of years:)